### DISCUSSION TITLE
gmod-development

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#19] [Blog comments]
# gmod-development

UPD 2022.06.19: В посте большая часть посвящена SublimeText и его плагинам, но спустя несколько лет его использования я с какой-то десятой попытки все-таки перешел на VSCode и жалею, что не сделал этого раньше.

https://blog.amd-nick.me/gmod-development

**Comment by shockpast** [comment]
Почему аналогичный плагин для *глуа* не от WilliamVenner (Billy), а от aStonedPenguin, он же всяко лучше него будет (imo).

**Reply by AMD-NICK** [reply]
Не понял тебя. Это ты на что написал?

**Reply by shockpast** [reply]
aStonedPenguin уже не обновляет своё [расширение](https://marketplace.visualstudio.com/items?itemName=aStonedPenguin.glua).
WilliamVenner обновляет своё [расширение](https://marketplace.visualstudio.com/items?itemName=venner.vscode-glua-enhanced) по сей день.

Так-вот, почему в посте ссылается на плагин от пингвина, если его уже не обновляют, ну уж очень давно((

**Reply by AMD-NICK** [reply]
> Так-вот, почему в посте ссылается на плагин от пингвина

Для меня решающим фактором было неудобное автодополнение Lua функций. Писал об этом [тут](https://t.me/c/1473957119/146454). Для просмотра может потребоваться [инвайт](https://t.me/+_RwEbmqxRWkyNWZi)
